### PR TITLE
[Notifier ] Add Discord notifier

### PR DESCRIPTION
**Pull request by connorhu**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      |  no
| New feature?  | no
| Deprecations? | no
| Tickets       | 
| License       | MIT
| Doc PR        | symfony/symfony-docs#13558

It finish the PR #36475

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Discord/DiscordOptions.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+namespace Symfony\Component\Notifier\Bridge\Discord;
+
+use Symfony\Component\Notifier\Bridge\Discord\Embeds\DiscordEmbedInterface;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Karoly Gossler <connor@connor.hu>
+ *
+ * @experimental in 5.1
```
Comment (hhamon) [inline src/Symfony/Component/Notifier/Bridge/Discord/DiscordOptions.php:21]: Should be `5.2` ?

**Comment by nicolas-grekas** [comment]
Could you please rebase and squash the PR?

**Comment by connorhu** [comment]
@nicolas-grekas i've done the rebase and now fabbot says "code looks good" (and the conflict came back).

**Comment by nicolas-grekas** [comment]
To fix the conflict, you need to 1. fetch the upstream symfony/symfony repo 2. rebase on the 5.x branch of this repo. Please also squash so that the PR contains only one commit (or two if there are two authors).

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @connorhu.

**Pull request merged**